src/crimson/osd/osd.cc

```diff
@@ -168,9 +168,9 @@ seastar::future<> OSD::open_meta_coll()
 
 seastar::future<OSDMeta> OSD::open_or_create_meta_coll(FuturizedStore &store)
 {
-  return store.open_collection(coll_t::meta()).then([&store](auto ch) {
+  return store.get_local_shard_store().open_collection(coll_t::meta()).then([&store](auto ch) {
```

Comment:
Worth a TODO:
// TODO: encapsulate into OSDMeta to use its FuturizedShardStore

Or fix it directly in this PR if convenient.